scripts/qa_crowbarsetup.sh

```diff
@@ -2341,6 +2341,16 @@ function addfloatingip()
     nova add-floating-ip "$instanceid" "$floatingip"
 }
 
+# by setting --dns-nameserver for subnet, docker instance gets this as
+# DNS info (otherwise it would use /etc/resolv.conf from its host)
+function adapt_dns_for_docker()
+{
+    # DNS server is the first IP from the allocation pool, or the
+    # second one from the network range
+    local dns_server=`neutron subnet-show fixed | grep allocation_pools | cut -d '"' -f4`
+    neutron subnet-update --dns-nameserver "$dns_server" fixed
```

Comment:
Don't we need just a very bit of error handling here?

If the query of the DNS server cloud result in an error message as described, then the variable would be empty, right? What would be result, if you run this command with an empty `$dns_server` variable? It should at least error out, to allow qa_crowbarsetup to fail.